### DISCUSSION TITLE
Display LCC as an extra "module"?

### DISCUSSION THREAD

**Discussion by trypuz** [edmcouncil/fibo#1372] [General]
Related: #974

**Comment by ElisaKendall** [comment]
The Languages, Countries, and Codes (LCC) standard is an OMG standard that is not part of FIBO.  It could be included in the FIBO viewer if you like - it's also available from the OMG and I believe from data.world.

**Comment by ElisaKendall** [comment]
This means, however, if we choose to depend on other ontologies (like SKOS, Dublin Core, etc.), the emerging Standard Business Reporting Metamodel (SBRM), we will have a similar decision to make about them.

**Comment by VladimirAlexiev** [comment]
SKOS and DC are small and people are not very likely to click on SKOS and DC props.

If SBRM is large, I'd also suggest it should be included in the viewer.

Perhaps you need a new section "External ontologies" or a new colored dot to distinguish those.

**Comment by rivettp** [comment]
This is purely an issue with what is displayed in FIBO Viewer, right?
There will be a general problem in that such external ontologies have URIs not controlled by EDMC so URIs cannot be redirected to the EDMC/FIBO viewer site and URI. Unless FIBO viewer hacks any such external URIs which occur in FIBO to replace them with something under the edmcouncil.org domain. Which I'd say would be misleading and somewhat unethical.
The external ontology owners are likely to put in place their own viewing mechanism: GLEIF has already, and OMG is likely to.

**Comment by VladimirAlexiev** [comment]
You already display external terms (thanks!). The issue is whether you can somehow incorporate them in the left-hand navigation menu.

If you do, indicate them with a purple dot or something similar.

I understand the technical difficulties related to resolution. If you cannot incorporate them in the navigation without taking control of OMG domain resolution (that is infeasible), cancel this issue.
